Feature/submission string changes

Oliver-Zimmerman opened the pull request:
<!-- 
Thank you for supporting us with your Pull Request! 🙌 ❤️  
Before submitting, please take the time to check the points below and provide some descriptive information.
-->

## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [ ] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [x] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) (if applicable)
* [ ] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

## Description

-removed "final" comments causing issue with translation
-added relevant comments for some strings

Oliver-Zimmerman committed:
Submission string changes.

-Submission string changes. Strings marked final are finalized.
-Changed the string value of submission done fragment.

Oliver-Zimmerman committed:
Update submission done layout to match new strings

added title which was missing

Oliver-Zimmerman committed:
Correction to some of the provided strings.

Updated strings as requested.

Oliver-Zimmerman committed:
Merge branch 'dev' into feature/submission-string-changes

Oliver-Zimmerman committed:
Update strings.xml

merge conflict resolution

Oliver-Zimmerman committed:
Update strings.xml

Oliver-Zimmerman committed:
Update strings.xml

-removed "<!-- final -->" comments causing issue with translation
-added relevant comments for some strings

Oliver-Zimmerman committed:
Merge branch 'dev' into feature/submission-string-changes

harambasicluka commented on Corona-Warn-App/src/main/res/values/strings.xml:
```diff
@@ -446,92 +446,115 @@
-    <string name="submission_test_result_invalid_steps_invalid_heading">Ihr Test-Ergebnis</string> <!-- final -->
-    <string name="submission_test_result_invalid_steps_invalid_body">Es gab ein Problem bei der Auswertung Ihres Tests. Bitte kontaktieren Sie das Gesundheitsamt um Information zum weiteren Vorgehen zu erhalten. Bitte entfernen Sie den Test wieder aus der Corona-Warn-App, damit Sie bei Bedarf einen neuen Test hinterlegen können.</string> <!-- final -->
-    <string name="submission_test_result_invalid_remove_test_button">Test entfernen</string> <!-- final -->
+    <string name="submission_test_result_steps_added_heading">Test erfolgreich hinzugefügt</string>
```
please add the translation comments to all strings

Oliver-Zimmerman committed:
Update strings.xml

Added all the tagged comments after removal of 'final' comments in relation to submission flow.

Oliver-Zimmerman committed:
Merge branch 'dev' into feature/submission-string-changes
